lua/shipwright/transform/contrib/windows_terminal.lua

```diff
@@ -0,0 +1,85 @@
+--- windows terminal transform, expects a table in the shape:
+--
+-- @param colors {
+-- "name" : "Theme Name",
```

Comment:
This  `@param` doc should be a "lua table", so it should instead be `name = "Theme Name",`, etc for all the keys.